packages/react-components/src/components/Picker/Trigger.tsx

```diff
@@ -86,6 +88,7 @@ export const Trigger: React.FC<ITriggerProps> = ({
       className={mergedClassNames}
       onClick={handleTriggerClick}
       tabIndex={0}
+      data-testid={dataTestId}
```

Comment:
As I wrote above, if we use `data-testid`, here we can simply do spread it by `{...props}`.